ofrak_patch_maker/ofrak_patch_maker/toolchain/utils.py

```diff
@@ -70,6 +78,34 @@ def get_repository_config(section: str, key: Optional[str] = None):
         )
 
 
+def get_exec_from_config(section: str, key: str):
+    """
+    Get an executable name or path from a toolchain.conf. Output instructions for configuring
+    toolchain paths if the exectuable is not available.
+
+    :param section: section name in config file
+    :param key: key in `config[section]`
+
+    :raises SystemExit: If `config[section]` or `config[section][key]` not found.
+    :return Union[str, List[Tuple[str, str]]]: the result of ``config.get(section, key)`` or
+        ``config.items(section)``
+    """
+
+    exec_path = get_repository_config(section, key)
+    if shutil.which(exec_path) is not None:
```

Comment:
I don't understand why we need `shutil.which` here when all of the paths in the configuration file are absolute and fully qualified?